src/ui/public/kbn_top_nav/kbn_top_nav_controller.js

```diff
@@ -83,7 +83,7 @@ export function KbnTopNavControllerProvider($compile) {
       if (menuItem.disableButton()) {
         return false;
       }
-      menuItem.run(menuItem, this, event.target);
+      menuItem.run(menuItem, this, get(event, 'target'));
```

Comment:
Might be worth adding a comment about why event might be undefined, as this was surprising to me.